apps/web/pages/_app.tsx

```diff
@@ -7,7 +9,17 @@ import "../styles/globals.css";
 function MyApp(props: AppProps) {
   const { Component, pageProps } = props;
   if (Component.PageWrapper !== undefined) return Component.PageWrapper(props);
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+        />
+      </Head>
```

Comment:
@PeerRich I think this should be in the PageWrapper Component in order to get the meta tag in all pages.

Only the calendar-cache page reach this code and this meta tag is useless there.

Reply:
I have already did this change, I think the PR title must change

<img width="892" alt="imagen" src="https://github.com/calcom/cal.com/assets/525369/8d523a53-7fe8-4366-8605-35f36c7d361c">